packages/rocketchat-ui-message/client/message.js

```diff
@@ -1,7 +1,31 @@
-/* globals renderEmoji renderMessageBody */
+/* globals pdfjsLib renderEmoji renderMessageBody */
 import _ from 'underscore';
 import moment from 'moment';
 
+pdfjsLib.GlobalWorkerOptions.workerSrc = '/packages/kb0304_pdfjs/build/pdf.worker.js';
```

Comment:
This path can cause some troubles if Rocket.Chat is running under a root URL with a subdirectory.